src/constants/site.ts

```diff
@@ -0,0 +1 @@
+export const PRODUCTION = 'mbtashutdowns.info';
```

Comment:
I think @lawrencet1 wanted us to also have a TransitMatters subdomain as well as this one? @idreyn can we replicate what we did for the Regional Rail Explorer where we have both?

Reply:
Is there another one for the regional rail explorer? If so I can't find a config, dns record or link to it

Reply:
There definitely was at one point, though I'm struggling to remember what it was - maybe we actually replaced it

Reply:
I don't recall us ever having an RRE subdomain

Reply:
Alright I say we just proceed with this then

Reply:
We had talked about shutdowns.transitmatters.org on the call.

Reply:
Yep we'll still add it, just need to figure out the right way to do it. Looking into it tonight